src/routers/alpha-router/gas-models/v3/v3-heuristic-gas-model.ts

```diff
@@ -70,25 +76,25 @@ export class V3HeuristicGasModelFactory extends IV3GasModelFactory {
         gasCostInToken: CurrencyAmount;
         gasCostInUSD: CurrencyAmount;
       } => {
-        const { gasCostInEth, gasUse } = this.estimateGas(
+        const { gasCostNativeCurrency, gasUse } = this.estimateGas(
           routeWithValidQuote,
           gasPriceWei,
           chainId
         );
 
-        const ethToken0 = usdPool.token0.address == WETH9[chainId]!.address;
```

Comment:
(re: my exclamation mark comment) oh I guess we were using exclamation marks before....? I'm not really a typescript extraordinaire yet, but I guess it must be ok?